tgui/packages/tgui/interfaces/DecalPainter.tsx

```diff
@@ -85,8 +85,8 @@ export const DecalPainter = (props) => {
             </Section>
           </Stack.Item>
           <Stack.Item grow style={{ overflowY: 'auto', overflowX: 'hidden' }}>
-            <Section title="Decal Style" scrollable fill>
-              <Flex direction="row" wrap="wrap" align="fill" justify="fill">
+            <Section title="Decal Style" fill>
+              <Flex wrap="wrap">
```

Comment:
```
<Stack.Item grow>
  <Section title="Decal Style" fill scrollable>
```

Reply:
Odd, could have sworn during testing this wasn't working until I did the overflow. Now scrollable is working though.